### ISSUE TITLE
Resource Reload Failed after placing Supply Wagon

### ISSUE THREAD

**Issue by Iskandos** [ldtteam/minecolonies#5008]
### Minecolonies version # Update to the newest minecolonies and structurize alpha first and see if your issue might be already fixed!

Version: Minecolonies: 0.10.771 (for Minecraft 1.15.2)  /  Structurize: 0.10.313 (for Minecraft 1.15.2)

### Expected behavior

- The game loading screen ;)

### Actual behaviour

- The game was working perfectly until I placed a Supply Wagon. Then it crashed to the launcher with a "Resource Reload Failed" message. From then on, I am unable to launch Minecraft at all (unless I launch the default unmodded version -- but where's the fun in that? =P ) -- it does the usual file checks and such, the menu music begins to play, but then it fails just before the menu pops up with "Resource Reload Failed" and returns me to the launcher with a crash report.

- I have tried with and without additional resource packs. I usually have PureBDcraft 128x MC115 active, but removed it to see if that fixed it... it didn't =(

- I probably have an unusual collection of mods with this setup -- some are just longtime favorites, but mainly I wanted to have a little magical empire building with Minecolonies in the Overworld to support my adventures as I explored the LotR dimension. ^_^  So it's possible it could be a conflict generated by another mod. I am unable to determine this tho -- all I know is that everything worked right up until I clicked the green checkbox on the Supply Wagon placement menu. 

- If I can provide any additional info, just let me know! ^_^

### Steps to reproduce the problem

1. Place a Supply Wagon.
2. Watch game crash. =(
3. Restart game from launcher and it is stopped with the Resource Reload Failed error.
4. ... Profit

### Logs: Latest.log and crashlog at https://gist.github.com/

- latest.log:  https://gist.github.com/Iskandos/9acdeb62f24384d9f7d5acca74e23f17
- crashlog:  https://gist.github.com/Iskandos/73b0d96c8708d634af03eed3fdde995f

**Comment by Iskandos** [comment]
I took some time to pull out all the mods and add them back in one by one today, and it looks like the issue is connected to Geolosys -- which is... odd... since Geolosys and Minecolonies seem to be completely unrelated in functionality. 

And they seemed to work quite well together right up until I used the Supply Wagon (I had spent a couple of days inworld before even building the Supply Wagon, just to get some things set up and do some location scouting, all without issue)

The whole purpose of Geolosys is to add block generation of mod-related ores -- few, if any, of those mods are at 1.15.2 atm, so I thought this would be a handy way to "future proof" my worlds... if/when those mods are updated to 1.15, I would have a stash of resources for them right from the start, all from the existing worldgen. But it seems that cunning plan may not be working so well :(

Well, despite all that, I still have no idea what the actual issue is, or which mod is causing it ^_^  I'm starting to wish I had just stayed with 1.7.10 -- I never had any trouble with that set haha! =P 

So... should I close out this ticket and start one with Geolosys instead? 

Or do you want to take a look and see if something stands out first?

**Comment by Iskandos** [comment]
hmm, well it seems that even without Geolosys installed I still get a crash when placing the Supply Wagon (in a fresh new world, not the same as the initial one) -- tho the game at least still functions afterwards now. Unfortunately, the building isn't generated, and an attempt to place the Wagon again just ends with another crash :(

Here's the crash reports from the first error:
https://gist.github.com/Iskandos/ab1f272e424e921268cf50c4fcb7a5ef

And here's the report from the second attempt:
https://gist.github.com/Iskandos/6097a942f1f375f234085396f267ad1e

**Comment by Iskandos** [comment]
The separate/standalone Structurize mod is giving me the same crash as when I place the Supply Wagon in Minecolonies. I decided to play around with it some in Creative to see what it could do (I was a bit limited with the Wiki being down, but forged ahead regardless =P). I copied a test building and saved it without issue. placed a duplicate of the test building, adjusted its position and rotation, and as soon as I hit the green check to commit... the game crashed, exactly the same way as when I place a Supply Wagon =(

Here's the report from the Structurize crash:
https://gist.github.com/Iskandos/23a13ea2831bfe14552749e359c71b3b

On a positive note, I really like the Structurize UI -- it's much more intuitive and user friendly than WorldEdit! =)  Hopefully whatever is causing this placement issue isn't too troublesome to track down and fix!
